go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/awslabs/goformation/v4 v4.15.6
 	github.com/buger/goterm v1.0.0
 	github.com/cnabio/cnab-to-oci v0.3.1-beta1
-	github.com/compose-spec/compose-go v0.0.0-20210722130045-6e1e1c2b26de
+	github.com/compose-spec/compose-go v0.0.0-20210729053941-ef8509c7a5eb
```

Comment:
This needs to be updated now that https://github.com/compose-spec/compose-go/pull/165 has been merged.